pkg/service/group_service_factory.go

```diff
@@ -16,63 +16,64 @@ type NsGroupService struct {
 	client *client.GroupMgmtClient
 
 	// Declare all the supported services
-	accessControlRecordService        *AccessControlRecordService
```

Comment:
I wonder why the order changed.  I feel like this happened before, too.  It makes it difficult to tell if there are any impactful changes here.